src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -10,8 +10,15 @@ import {
 // custom error message component that renders the correct font according to the design system guidelines
 const ErrorMessage = styled(AscErrorMessage).attrs(() => ({
   role: 'alert',
-}))`
+}))<{ invalid: boolean }>`
   margin-top: 0;
+  white-space: pre-wrap;
+  display: none;
+  ${({ invalid }) =>
+    invalid &&
+    css`
+      display: flex;
+    `}
```

Comment:
This change won't solve the issue at hand; the point is that the element that holds the error message, should be present in the DOM. Setting its `display` value to 'none' will effectively hide it for browsers and screen readers which defeats the purpose.
See https://www.w3.org/WAI/WCAG21/working-examples/aria-role-status-searchresults/ for an example.